app/helpers/application_helper/toolbar/saved_reports_center.rb

```diff
@@ -1,7 +1,7 @@
 class ApplicationHelper::Toolbar::SavedReportsCenter < ApplicationHelper::Toolbar::Basic
   button_group('saved_report_reloading', [
     button(
-      :reload,
+      :miq_report_saved_reports_reload,
```

Comment:
This is for the saved reports button.

Left: admin view
Right:  non-admin view with the button missing (before this change)
![image](https://user-images.githubusercontent.com/19339/118854369-aa852f80-b8a2-11eb-88e6-de9fa7799cc3.png)

Reply:
Can we just piggyback on `miq_report_saved_reports_view` ?  It seems weird to give a dedicated feature to the reload button, unless it's actually like re-executing the reports under the covers.

Reply:
Good question. I ask that about lots of features.  I don't know why we have explicit copy vs. edit permissions on chargeback rates, reports, etc.

Others have `_reload` so that's the pattern.  I don't know that it's necessary:

```
db/fixtures/miq_product_features.yml:      :identifier: miq_request_reload
db/fixtures/miq_product_features.yml:      :identifier: ae_miq_request_reload
db/fixtures/miq_product_features.yml:      :identifier: cloud_volume_reload
db/fixtures/miq_product_features.yml:      :identifier: miq_task_reload
db/fixtures/miq_product_features.yml:    :identifier: perf_reload
db/fixtures/miq_product_features.yml:        :identifier: vm_perf_reload
```

Reply:
ok, let's follow the pattern for now and remove in the future if not needed.